src/main/java/com/mraof/minestuck/world/lands/GristLayerInfo.java

```diff
@@ -90,8 +91,12 @@ private GristLayerInfo(long seed, GristType baseType)
 	
 	public GristType randomTypeFor(LivingEntity entity)
 	{
-		BlockPos pos = entity.blockPosition();
-		return randomLayer(entity.getRandom()).getTypeAt(pos.getX(), pos.getZ());
+		if(MinestuckConfig.SERVER.hardMode.get() && entity.getRandom().nextBoolean()) {
+			return GristTypes.ARTIFACT.get();
+		} else {
+			BlockPos pos = entity.blockPosition();
+			return randomLayer(entity.getRandom()).getTypeAt(pos.getX(), pos.getZ());
+		}
```

Comment:
Would you mind adding a comment, does this mean that there is a chance for artifact underlings to appear in hardmode?